meal_mate/app/src/main/java/com/android/engineer/mealmate/ui/theme/Color.kt

```diff
@@ -13,6 +13,7 @@ val Pink40 = Color(0xFF7D5260)
 val OrangePrimary = Color(0xFFd17e63)
 val OrangeOnPrimary = Color(0xFFf3e5de)
 val OrangeOnBackground = Color(0xFFe8b796)
+val OrangeDark = Color(0xFF705744)
```

Comment:
@Siduchels  Just curios.. did you use the default color scheme's or did you make any changes to the theme?

Reply:
@nasreekar For now, we’re not using theming for colors; instead, we’re directly referencing colors from color.kt. Most of the UI components we’ve created are generic, so we’re calling colors directly from there.